components/molecules/ContributionsEvolutionByTypeCard/contributions-evolution-by-type-card.tsx

```diff
@@ -1,63 +1,38 @@
-import { useState } from "react";
 import { ResponsiveLine } from "@nivo/line";
 import { format } from "date-fns";
-import Button from "components/atoms/Button/button";
+import { useMemo } from "react";
 import Card from "components/atoms/Card/card";
-import Icon from "components/atoms/Icon/icon";
-import {
-  DropdownMenu,
-  DropdownMenuContent,
-  DropdownMenuItem,
-  DropdownMenuTrigger,
-} from "components/atoms/Dropdown/dropdown";
-import PeopleIcon from "img/icons/people.svg";
-import CalendarIcon from "img/calendar.svg";
-import ChevronDownIcon from "img/chevron-down.svg";
-import SVGIcon from "components/atoms/SVGIcon/svg-icon";
 
-const dataTypes = ["active", "new", "churned"] as const;
+const dataTypes = ["active", "new", "alumni"] as const;
 type Stat = (typeof dataTypes)[number];
 
 const dataLabels = {
   active: "Active",
   new: "New",
-  churned: "Churned",
+  alumni: "Alumni",
 } as const satisfies Record<Stat, string>;
 
 const colors = {
   active: "#46a758", // green
   new: "#0ea5e9", // blue
-  churned: "#f59e0b", // orange
+  alumni: "#f59e0b", // orange
 } as const satisfies Record<Stat, string>;
 
 export interface ContributionEvolutionByTypeDatum {
-  startTime: string;
+  time_start: string;
+  time_end: string;
   active: number;
   new: number;
-  churned: number;
+  alumni: number;
+  all: number;
 }
 
-interface Props {
-  data: ContributionEvolutionByTypeDatum[];
+interface ContributionsEvolutionByTypeProps {
+  data?: ContributionEvolutionByTypeDatum[];
+  isLoading: boolean;
 }
 
-const dateFilters = {
```

Comment:
These filters are no longer necessary. There is a global range filter and the other filter was not supposed to be included as we're showing all contributor types in the graph all the time.